CONTRIBUTING.md

```diff
@@ -1,193 +1,49 @@
-# Developer Introduction
+# Contributing to ProbNum
 
-Contributions to ProbNum are very welcome. Before getting started make sure to read the following guidelines.
+Contributions to ProbNum are very welcome! Before getting started make sure to read the following guidelines.
 
-All contributions to ProbNum should be made via pull requests (PR) to the
-[master branch](https://github.com/probabilistic-numerics/probnum/tree/master) on GitHub. Some suggestions for
-a good PR are:
+Some suggestions for contributions:
 
-- implements or fixes one functionality;
-- includes tests and appropriate documentation; and
-- makes minimal changes to the interface and core codebase.
+- adding documentation
+- adding tutorials
+- requesting a feature
+- implementing a new feature
+- reporting a bug
+- fixing a bug
+- adding missing tests
 
-If you would like to contribute but are unsure how, then writing examples, documentation or working on
-[open issues](https://github.com/probabilistic-numerics/probnum/issues) are a good way to start. See the
-[developer guides](https://probnum.readthedocs.io/en/latest/development/developer_guides.html)
-for detailed instructions.
+If you would like to contribute but are unsure how, then writing turorials, documentation or working on
+[open issues](https://github.com/probabilistic-numerics/probnum/issues) are a good way to start. 
 
-## Getting Started
+## Reporting a bug
+If you find a bug, please report it by opening an issue on the GitHub page, and tag it as a bug.
+Reproducibility is key, so if possible provide a working code snipped that produces the bug, and tell us the python 
```

Comment:
snippet that reproduces the bug and report the Python and ProbNum version you are using.

Reply:
done, I used mention instead of report to not repeat the same verb.